common/host_util.go

```diff
@@ -30,3 +32,19 @@ func GetLocalIp() string {
 	localIp, _ = gxnet.GetLocalIP()
 	return localIp
 }
+
+func HandleRegisterIPAndPort(url *URL) {
+	// if developer define registry port and ip, use it first.
+	if ipToRegistry := os.Getenv("DUBBO_IP_TO_REGISTRY"); len(ipToRegistry) > 0 {
+		url.Ip = ipToRegistry
+	}
+	if len(url.Ip) == 0 {
+		url.Ip = GetLocalIp()
+	}
+	if portToRegistry := os.Getenv("DUBBO_PORT_TO_REGISTRY"); len(portToRegistry) > 0 {
```

Comment:
Please create a const for `DUBBO_PORT_TO_REGISTRY` in [common/constant/env.go](https://github.com/apache/dubbo-go/blob/master/common/constant/env.go).